maven/lib/dependabot/maven/version.rb

```diff
@@ -138,7 +136,7 @@ def convert_dates(version, other_version)
       end
 
       def split_into_prefixed_tokens(version)
-        ".#{version}".split(/(?=[\-\.\+])/)
+        ".#{version}".split(/(?=[\-\.])/)
```

Comment:
This is from the above spec too. Maven does not special case the plus sign or consider build identifiers.